content/applications/hr/fleet/service.rst

```diff
@@ -86,61 +298,219 @@ clicked, sort the data in different ways.
+services with the selected status appear in the column.
+
+.. image:: service/kanban-status.png
+   :align: center
+   :alt: The oil change service records showing only repairs with past-due activities.
+
+.. _service/schedule-activity:
+
+Schedule activities
+*******************
+
+To schedule an activity for a repair or service from the Kanban view, click the :guilabel:`activity
+icon` in the lower-right corner of the service record, and click :guilabel:`+ Schedule an activity`.
+A :guilabel:`Schedule Activity` pop-up window appears.
+
+Enter the following information on the form:
+
+- :guilabel:`Activity Type`: using the drop-down menu, select the activity being scheduled. The
+  default options are :guilabel:`Email`, :guilabel:`Call`, :guilabel:`Meeting`, :guilabel:`To-Do`,
+  and :guilabel:`Upload Document`.
+- :guilabel:`Summary`: enter a short description of the activity, such as `Schedule oil change`.
+- :guilabel:`Due Date`: using the calendar popover, select the date the activity must be completed.
+  Using the :guilabel:`< (left)` and :guilabel:`> (right)` arrow icons, navigate to the desired
+  month, then click on the :guilabel:`date` to select it.
+- :guilabel:`Assigned to`: using the drop-down menu, select the user responsible for the activity.
+- :guilabel:`Notes`: add any notes or details in the blank area in the bottom-half of the form.
+
+When the :guilabel:`Schedule Activity` is completed, click :guilabel:`Schedule` to schedule the
+activity, or click :guilabel:`Done & Schedule Next` to schedule the current activity and schedule
+another activity for the same repair.
+
+.. seealso::
+   For more detailed information regarding activities, refer to the main :doc:`activities
+   <../../essentials/activities>` document.
+
+Add a service
+*************
+
+A new repair can be added from this view. Click the :guilabel:`+ (plus icon)` in the top-right
+corner of the Kanban column and a new block appears at the top of the column, beneath the Kanban
+title.
 
-All the services with the same status appear in the corresponding column, from :guilabel:`Running`
-to :guilabel:`Cancelled`. To change the status of a service, simply click and drag the service card
-to the desired stage.
+Enter a :guilabel:`Name` for the service or repair, then click :guilabel:`Add`. A :guilabel:`Create`
+service form appears in a pop-up window. :ref:`Enter all the information <service/service-form>` on
+the service form, then click :guilabel:`Save & Close` to add the record. The new record now appears
+in the Kanban column.
 
 Graph view
 ~~~~~~~~~~
 
-Another way to view the data is in a graph. To view the graph, click the :guilabel:`Graph icon`,
-which is the third icon in the top right, and appears as a small graph.
+Another way to view the data is in a graph. To change to the graph view, click the :guilabel:`Graph
+icon`, which is the third icon in the top right, and appears as a small graph.
```

Comment:
I'm pushing back on this one again, since when you rollover it, it does say Graph- so I feel that is the "proper name" of the actual button.  But I did take 'icon' out of the gui since that does not appear on roll-over, and is not part of the proper name.